docs/mnist_example/README.md

```diff
@@ -195,10 +195,19 @@ int main() {
   // The first input is of tensor<1x1x28x28xf32>.
   int64_t rank = 4;
   int64_t shape[] = {1, 1, 28, 28};
-  OMTensor *tensor = omTensorCreate(img_data, shape, rank, ONNX_TYPE_FLOAT);
-  // Create a tensor list.
+
+  // Create a tensor using omTensorCreateWithOwnership (returns a pointer to OMTensor).
+  // When the parameter, owning is set to "true", the OMTensor will release the data
+  // pointer upon destruction. If owning is set to false, the data pointer will not be
+  // freed upon destruction.
+  OMTensor *tensor = omTensorCreateWithOwnership(img_data, shape, rank, ONNX_TYPE_FLOAT, /*owning=*/true);
+
+  // Create a tensor list using omTensorListCreateWithOwnership (returns a pointer to the OMTensorList).
+  // When the parameter, owning is set to "true", the OMTensorList will release the data
```

Comment:
In situations like this it would be best for me to have a meeting to discuss instead of writing in a PR, I process data better via audio. I spoke with Jan and he stated the following  "I think the word data_ptr adds confusion, because it is used everywhere to describe the C arrays that hold the actual data. For the TensorList, it's just a pointer to an array, which contains pointers to the tensors." So I will update the wording to avoid confusion as Jan mentioned.

Reply:
Updated...sending PR link to Jan as well to see if this is clearer.